JsonFx/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+export DOTNET_CLI_TELEMETRY_OPTOUT=1
+
+set -x #echo on
+
+rm -rf */bin */obj
+
+dotnet build -c Release
```

Comment:
I noticed that on these new project builds we are always building in release mode. Is that anything, like conditionally compiled code, that could get missed because it's never built in debug mode?

Reply:
Correct me if I'm wrong, but if we have code that would be conditionally compiled in only during debug mode, that part is always compiled out in the DLLs built using release mode. That code won't be there even if the DLL is used in another project and that project runs in debug mode.

The project itself might not run in debug mode, and we should be worried about that, but the release is going to be fine.

Reply:
Yeah, release will be fine, I was just thinking about dev work.